pkg/build/api/validation/validation_test.go

```diff
@@ -105,8 +105,18 @@ func TestBuildConfigValidationFailure(t *testing.T) {
 				DockerImageReference: "repository/data",
 			},
 		},
+		Triggers: []buildapi.BuildTriggerPolicy{
+			{
+				Type:        buildapi.ImageChangeBuildTriggerType,
+				ImageChange: &buildapi.ImageChangeTrigger{},
+			},
+			{
+				Type:        buildapi.ImageChangeBuildTriggerType,
+				ImageChange: &buildapi.ImageChangeTrigger{},
+			},
+		},
 	}
-	if result := ValidateBuildConfig(buildConfig); len(result) != 1 {
+	if result := ValidateBuildConfig(buildConfig); len(result) != 2 {
```

Comment:
Yeah, that was my laziness, but you're right this needs better, iow. exact error checking like in other tests. Will update in a follow up.